src/Core/Models/Business/TaxInfo.cs

```diff
@@ -37,12 +37,24 @@ public string TaxIdType
 
             switch (BillingAddressCountry)
             {
+                case "AD":
+                    _taxIdType = "ad_nrt";
+                    break;
                 case "AE":
```

Comment:
There's a repeated value for the country code "AF" - it has the tax ID type "ae_trn", which appears to be the same as "AE". Please, ensure this isn't a mistake.